oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStore.java

```diff
@@ -2640,20 +2687,25 @@ private void invalidatePaths(@NotNull Set<Path> pathsToInvalidate) {
             // nothing to do
             return;
         }
+        invalidatePaths0(pathsToInvalidate, "split",
+                " Will be retried with next background split operation.");
+    }
+
+    private void invalidatePaths0(@NotNull Iterable<Path> pathsToInvalidate, String type,
```

Comment:
this cries for a better method name :-)

Reply:
`invalidatePaths1` ? ;)

Reply:
(open for suggestions!)

Reply:
What about `createInvalidationJournalEntry`?

Reply:
thx, renamed in https://github.com/apache/jackrabbit-oak/pull/1284/commits/e0bd8a1c1ec567ab551c787fea9f67a052fe6b44